README.md

```diff
@@ -45,20 +48,21 @@ instructions under "Running Old Versions" to display the old guides.
 After cloning this repo, install dependencies and run the demo app:
 
 ```bash
-$ npm install
-$ ember serve
+npm install
+bower install
```

Comment:
Only one of the two lines above are needed depending on how ember-charts is installed, correct? Maybe worth adding a comment to side of this line